client/my-sites/plugins/plugins-dashboard/index.tsx

```diff
@@ -259,14 +259,7 @@ const PluginsDashboard = ( {
 			return;
 		}
 
-		doActionOverSelected(
-			'updating',
-			( siteId: number, plugin: Plugin ) => {
-				handleUpdatePlugins( [ plugin ], updatePluginAction, [] );
-				removePluginStatuses();
```

Comment:
This action does not seem to be required, removing it does not seem to break anything.

Reply:
Perhaps the intention was to remove the status of the previous action before dispatching a new one

![CleanShot 2025-01-24 at 19 42 57](https://github.com/user-attachments/assets/4c47fe63-cef6-4106-b1e2-3d8f72a6ad8b)

I think I like all the statuses displayed though as if someone clicks "Update" to multiple plugins they will end up with only the status of the last plugin clicked.